caom2-artifact-sync/src/main/java/ca/nrc/cadc/caom2/artifactsync/Main.java

```diff
@@ -360,7 +388,9 @@ private static void usage() {
         sb.append("\n     --artifactStore=<fully qualified class name>");
         sb.append("\n     --database=<server.database.schema>");
         sb.append("\n     --collection=<collection> (currently ignored)");
-        sb.append("\n     --mode=[dual | harvest | download] : Operate in both harvest and download mode (Default) | ");
+        sb.append("\n     --tap=<tapResourceID> (required by validate mode)");
+        sb.append("\n     --reportOnly (prints validation summary only, does not update artifact skip uri table)");
+        sb.append("\n     --mode=[dual | harvest | download | validate] : Operate in both harvest and download mode (Default) | ");
```

Comment:
Instead of the comment "Operate in both harvest and download mode", you could now say something like:  "The mode in which to run this tool.  'dual' is the combination of harvest and download modes"